package.json

```diff
@@ -14,18 +18,17 @@
   "author": "Arron Hunt",
   "license": "MIT",
   "dependencies": {
-    "@react-native-async-storage/async-storage": "^1.13.3",
-    "@taskade/eslint-plugin": "^0.2.0",
     "emoji-datasource": "^6.0.0"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/taskade/react-native-emoji-selector.git"
+  "peerDependencies": {
+    "@react-native-async-storage/async-storage": "^1.13.0",
+    "react-native": "^0.0.0-0 || 0.60 - 0.71 || 1000.0.0"
```

Comment:
nit: would `>0.64.0` work here?

the semver range can be tested with the [npm semver calculator](https://semver.npmjs.com/)